Assets/HoloToolkit/UX/Scripts/Dialog/DialogShell.cs

```diff
@@ -28,9 +28,6 @@ public class DialogShell : Dialog
         [SerializeField]
         private TextMesh messageText;
 
-        [SerializeField]
-        private GameObject[] oneButtonSet;
```

Comment:
@cre8ivepark or @johnppella, is this supposed to be used? Looks like it's not. Are there planned but unimplemented features here?